jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/logging/AnsiLoggerWithFooter.java

```diff
@@ -103,13 +116,16 @@ public void log(Level logLevel, String message) {
 
     singleThreadedExecutor.execute(
         () -> {
-          if (eraseFooter()) {
-            // If a previous footer was erased, the message needs to go up a line. However, we go up
-            // two lines (and then down one line by calling "accept()" once) to correct an issue in
-            // Maven: https://github.com/GoogleContainerTools/jib/issues/1952
+          boolean didErase = eraseFooter();
```

Comment:
L119-126 and L152-159 are the same, I wonder if we can pull those lines into `eraseFooter()`.

Reply:
Yeah, the code is duplicated, but I find the logic of the duped code is a bit weird to shove into `eraseFooter()` for the sake of removing redundancy. The caller is expected to follow up with doing the correct thing after calling `eraseFooter()`(i.e., must print the next message with the returned prefix), so I don't quite like it from the API perspective. I think this amount of dup is not worse than that.